expression-test/expression_test_parser.cpp

```diff
@@ -53,7 +53,7 @@ void writeJSON(rapidjson::PrettyWriter<rapidjson::StringBuffer>& writer, const V
 
 using ErrorMessage = std::string;
 using JSONReply = variant<JSDocument, ErrorMessage>;
-JSONReply readJson(const filesystem::path& jsonPath) {
+JSONReply readJson(const std::filesystem::path& jsonPath) {
     auto maybeJSON = util::readFile(jsonPath);
```

Comment:
You need to call `jsonPath.string()` here for `readFile`

Reply:
> Paths are implicitly convertible to and from [std::basic_string](https://en.cppreference.com/w/cpp/string/basic_string)s, which makes it possible to use them with other file APIs.

https://en.cppreference.com/w/cpp/filesystem/path

Reply:
OK just noticed it does not compile for Windows. 💢

Reply:
It's a bit weird but what I think is going on: The implicit conversion is to the system native type, string for POSIX and wstring for Win32. So an implicit conversion to `std::wstring` would compile on Windows.

Reply:
Yes makes sense.